aviary/wren/model.py

```diff
@@ -267,4 +267,4 @@ def forward(
         return cry_fea
 
     def __repr__(self) -> str:
-        return self.__class__.__name__
+        return f"{type(self).__name__}(elem_emb_len={self.elem_emb_len}, sym_emb_len={self.sym_emb_len})"
```

Comment:
... and here for Wren.

Reply:
we should put in the numbers of layers and heads per layer?

Reply:
would need to save some more variables in the class state but that's okay?

Reply:
Top level or put them in a `params`/`props` dict?